scripts/build_musl_static.sh

```diff
@@ -0,0 +1,129 @@
+#!/bin/sh
+
+set -eu
+
+
+MUSL_CFLAGS=${CFLAGS:-"-O3 -flto -g"}
+
+
+message()
+{
+    echo ""
+    echo "[+]"
+    echo "[+] $1"
+    echo "[+]"
+    echo ""
+}
+
+SCRIPT=$(readlink -f "$0")
+SCRIPTDIR=$(dirname "$SCRIPT")
+
+cd "${SCRIPTDIR}" || exit
+
+mkdir -p musl_static_builddir/
+
+cd musl_static_builddir/ || exit
+
+
+NCURSES_VERSION=6.2
+NCURSES_FILE=ncurses-${NCURSES_VERSION}.tar.gz
+
+if ! [ -e ${NCURSES_FILE} ]; then
+    message "Downloading ncurses v${NCURSES_VERSION}..."
+    wget https://ftp.gnu.org/pub/gnu/ncurses/${NCURSES_FILE}
```

Comment:
Please don't introduce any arbitrary random downloading into the build process.

Instead if this is missing fail with a meaningful error message.